v14.x/config.sh

```diff
@@ -0,0 +1,2 @@
+export LAYER_NAME=nodejs14
+export NODE_VERSION=14.3.0
```

Comment:
Latest is 14.15.1

Reply:
🤦 I looked that up too... Ok I updated it. After updating, I'm seeing something a little unexpected in the `test.sh` logs. They now include `ping` messages -- unknown source as of now.

<img width="1353" alt="Screen Shot 2020-12-11 at 7 37 56 PM" src="https://user-images.githubusercontent.com/15269623/101970619-a87abe80-3be8-11eb-9baa-2d6cfd67dd59.png">

Reply:
I found the source
```js
const interval = setInterval(console.log, 100, 'ping')
```
the question seems to be, why did I not get any ping logs before? 🤷‍♂️